docs/posts/deprecate-py-legacy.md

```diff
@@ -1,3 +1,9 @@
+---
+parent: posts
+date: 2024-10-28
+search_exclude: true
```

Comment:
Why exclude from search?

Reply:
everything under posts is excluded from search IIUC